Assets/Decontainment/Assembly/Program.cs

```diff
@@ -109,6 +111,11 @@ public struct ArgumentSpec
         {
             ArgumentSpec.BRANCH_LABEL
         };
+        public static readonly ArgumentSpec[] ONE_INPUT_CONTROL_FLOW_SPECS = new ArgumentSpec[]
```

Comment:
VAL1's description is "Left value", which doesn't make much sense for the random branching instruction's second argument. Perhaps remove this constant arg spec array and give the instruction a custom one. Arg spec 0 can still be BRANCH_LABEL, but spec 1 should probably have "Branch chance" as its description.